### ISSUE TITLE
MD034 Bare URL Quick Fix at end of sentence erroneously includes the period.

### ISSUE THREAD

**Issue by ewlloyd** [DavidAnson/vscode-markdownlint#115]
# Issue

URLs generally don't include trailing periods. Therefore, the Quick Fix for MD034 usually does the wrong thing for URLs at the end of a sentence.

# Current behavior:
```markdown
	More information can be found at https://example.com/moreInformation.html.
```
is quickfixed to:
```markdown
	More information can be found at <https://example.com/moreInformation.html.>
```

# Expected behavior:
```markdown
	More information can be found at https://example.com/moreInformation.html.
```
should be quickfixed to:
```markdown
	More information can be found at <https://example.com/moreInformation.html>.
```
Note that the period is outside the angle brackets.

**Comment by nschonni** [comment]
https://github.com/DavidAnson/markdownlint/issues/283

**Comment by DavidAnson** [comment]
Fixed in the library by https://github.com/DavidAnson/markdownlint/commit/6ac5fa52a6346d3398ce9ead5939dbc4727f137e for the next release.

**Comment by DavidAnson** [comment]
This should be fixed in Friday’s release.
